src/pages/SearchPage.tsx

```diff
@@ -12,42 +12,39 @@ import Navigation from '@libs/Navigation/Navigation';
 import * as OptionsListUtils from '@libs/OptionsListUtils';
 import Performance from '@libs/Performance';
 import * as ReportUtils from '@libs/ReportUtils';
+import type {OptionData} from '@libs/ReportUtils';
 import * as Report from '@userActions/Report';
 import Timing from '@userActions/Timing';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
-import personalDetailsPropType from './personalDetailsPropType';
-import reportPropTypes from './reportPropTypes';
-
-const propTypes = {
-    /* Onyx Props */
+import type * as OnyxTypes from '@src/types/onyx';
 
+type SearchPageProps = {
     /** Beta features list */
-    betas: PropTypes.arrayOf(PropTypes.string),
+    betas: OnyxEntry<OnyxTypes.Beta[]>;
 
     /** All of the personal details for everyone */
-    personalDetails: PropTypes.objectOf(personalDetailsPropType),
+    personalDetails: OnyxEntry<OnyxTypes.PersonalDetailsList>;
 
-    /** All reports shared with the user */
-    reports: PropTypes.objectOf(reportPropTypes),
+    /** The report currently being looked at */
+    reports: OnyxEntry<OnyxTypes.Report>;
 
     /** Whether we are searching for reports in the server */
-    isSearchingForReports: PropTypes.bool,
+    isSearchingForReports: OnyxEntry<boolean>;
 };
 
-const defaultProps = {
-    betas: [],
-    personalDetails: {},
-    reports: {},
-    isSearchingForReports: false,
+type SearchOptions = {
+    recentReports: OnyxTypes.Report[];
+    personalDetails: OnyxTypes.PersonalDetails[];
+    userToInvite: [];
```

Comment:
Just an empty array? it doesn't look right for the type, also are you sure it's array at all?